imports/plugins/core/orders/client/graphql/mutations/createRefund.js

```diff
@@ -0,0 +1,18 @@
+import gql from "graphql-tag";
+import { orderCommonFragment } from "../fragments/orderCommon";
+
+export default gql`
+  mutation createRefundMutation($amount: Float! , $orderId: ID!, $paymentId: ID!, $reason: String, $language: String! = "en") {
```

Comment:
I'm guessing `$language` was left here accidentally

Reply:
`language` is needed for `OrderCommon`. It was left here on purpose. Does it need to be added to the query then?

Reply:
For now I think this is fine. I didn't see it was used by the fragment.